app/models/user.rb

```diff
@@ -186,14 +186,14 @@ def self.from_omniauth(auth)
   # Handle user creation from provider
   # rubocop:disable Metrics/AbcSize
   def self.create_from_provider_data(provider_data)
-    user = User.find_or_initialize_by(email: provider_data.info.email)
+    email = provider_data.info.email.downcase
+    user = User.find_or_initialize_by(email: email)
```

Comment:
Since we are not using the email variable anywhere else you can have the downcase call in the same line where you find the user:

```
user = User.find_or_initialize_by(email: provider_data.info.email.downcase)
```

Reply:
Cool, that sounds good. I'll make the change.